core/controllers/topics_and_skills_dashboard.py

```diff
@@ -85,8 +85,8 @@ def get(self):
 
         untriaged_skill_summary_dicts = []
         for skill_summary_dict in skill_summary_dicts:
-            if (skill_summary_dict['id'] not in skill_ids_assigned_to_some_topic) and ( #pylint: disable=max-line-length
-                    skill_summary_dict['id'] not in skill_ids_for_unpublished_skills): #pylint: disable=max-line-length
+            if (skill_summary_dict['id'] not in skill_ids_assigned_to_some_topic) and ( #pylint: disable=line-too-long
```

Comment:
No, please do not use pylint pragmas -- this can be broken into multiple lines! Ditto below. Thanks!

Reply:
I tried breaking it up but can't figure out how without introducing syntax errors :\ Sorry not too familiar with python syntax